OIIInspector/LocalExecutor.py

```diff
@@ -0,0 +1,50 @@
+import subprocess
+import logging
+import shlex
+
+
+LOG = logging.getLogger("OIIInspector")
+
+
+class LocalExecutor(object):
```

Comment:
I think this was inspired by existing code, but I don't know if object-oriented design is necessary here. Maybe you could just add a method for running a command to the OIIIClient class that would handle all this stuff

Reply:
I now import just method (not a class as it was before last push) from file called LocalExecutor, it seems to me to be a more readable and clearer code, but I can move that method to OIIIClient too, If you consider it to be a better solution.